src/client/components/unauthorized_page/signIn.tsx

```diff
@@ -0,0 +1,108 @@
+import React, {Component} from 'react';
+import './singin.css';
+
+/**
+ * this sign in form will re-direct to narrative only
+ */
+export class SignIn extends Component {
+
+    redirect(param:string){
+      let ele = document.getElementById(param);
+      let form = document.getElementById('login-form') as HTMLFormElement;
+      if(ele && form){
+        ele.click();
+        form.submit();
+      }
+    }
+
+    render(){
+     return (
+     <>
+        <div className="sign-in-container">
+        <h1>Sign in with...</h1>
+					<img style={{marginTop: '1em'}} src='http://kbase.us/wp-content/uploads/2014/11/kbase-logo-web.png'/>
+					<div className='sign-up-botton-container'>
+						<form className='login-form' id='login-form'
+							action="https://kbase.us/services/auth/login/start"
+							method="post">
+							<div className='sign-up-botton-spacer'>
+								<div className='sign-up-botton'>
+									<svg className="logo" id='google-logo' width="45px" height="45px" viewBox="0 0 18 18" role="presentation" aria-hidden="true" focusable="false">
```

Comment:
I think it would be cleaner to just put the svg in static/images and load it with an img tag here so we don' thave to have all this svg code in our TS

Reply:
help